subcommand/get-consul-client-ca/command.go

```diff
@@ -0,0 +1,157 @@
+package getconsulclientca
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/command/flags"
+	"github.com/hashicorp/go-hclog"
+	"github.com/mitchellh/cli"
+)
+
+type Command struct {
+	UI cli.Ui
+
+	flags *flag.FlagSet
+
+	flagOutputFile    string
+	flagHttpAddr      string
+	flagCAFile        string
+	flagTLSServerName string
+	flagLogLevel      string
+
+	once sync.Once
+	help string
+}
+
+func (c *Command) init() {
+	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
+	c.flags.StringVar(&c.flagOutputFile, "output-file", "",
+		"The path to the file where to put the Consul client's CA certificate.")
+	c.flags.StringVar(&c.flagHttpAddr, "http-addr", "",
+		"The HTTP address of the Consul server. This can also be provided via the CONSUL_HTTP_ADDR environment variable.")
+	c.flags.StringVar(&c.flagCAFile, "ca-file", "",
+		"The path to the CA file to use when making requests to the Consul server. This can also be provided via the CONSUL_CACERT environment variable")
+	c.flags.StringVar(&c.flagTLSServerName, "tls-server-name", "",
+		"The server name to set as the SNI header when sending HTTPS requests to Consul. This can also be provided via the CONSUL_TLS_SERVER_NAME environment variable.")
```

Comment:
I decided not to reuse consul's HTTP flags package here since we're not supposed to be using it.